code/modules/hydroponics/beekeeping/beebox.dm

```diff
@@ -13,12 +13,12 @@
 
 
 /mob/living/carbon/human/bee_friendly()
-	if(dna && dna.species && dna.species.id == SPECIES_PODPERSON) //bees pollinate plants, duh.
+	if(dna && dna.species && ispodperson(src)) //bees pollinate plants, duh.
```

Comment:
```suggestion
	if(ispodperson(src)) //bees pollinate plants, duh.
```

Reply:
Or you could make it a trait or something

Reply:
Ill just keep it at ispodperson(), making it a trait would require touching more things like bee AI